### PR TITLE
Added Elu activation support for untyped tensors

### PR DESCRIPTION
**Pull request by championballer**
Need ELU activation for reinforcement learning experimentation. Have personal experience with good performance improvement in DQN.

### CODE REVIEW

File: hasktorch/src/Torch/Functional.hs [hunk 1]
```diff
@@ -289,6 +289,20 @@ relu
...
+elu t = unsafePerformIO $ (cast1 ATen.elu_t) t
+
+-- | Applies Exponential linear unit function element-wise, with alpha input, \(\text{ELU}(x) = \max(0,x) + \min(0, \alpha * (\exp(x) - 1))\)
+elu_a
+    :: Scalar s 
+    => Tensor -- ^ input
+    -> s -- ^ alpha value for ELU formulation
+    -> Tensor -- ^ output
+elu_a t a = unsafePerformIO $ (cast2 ATen.elu_ts) t a
+
```
Comment (junjihashimoto) [inline hasktorch/src/Torch/Functional.hs]: For now, we use suffix of `'` with default argument.
I think it is better to put input tensor into last argument like following code.

```haskell
elu :: Scalar s => s -> Tensor -> tensor
elu' :: Tensor -> Tensor
```

Comment (championballer) [reply hasktorch/src/Torch/Functional.hs]: Hey, yeah. I was going through the currying conversation on slack today and realised. Will update this asap.

Comment (championballer) [reply hasktorch/src/Torch/Functional.hs]: @junjihashimoto Please have a look now?

Comment (junjihashimoto) [reply hasktorch/src/Torch/Functional.hs]: Thx!
There is no test of elu on both untyped and typed spec.
Could you add the tests of elu?

Comment (championballer) [reply hasktorch/src/Torch/Functional.hs]: Sure.

Comment (championballer) [reply hasktorch/src/Torch/Functional.hs]: @junjihashimoto This is ready for review.

**Comment by junjihashimoto** [comment]
LGTM!

**Pull request merged**